src/components/Dropzone.tsx

```
@@ -63,31 +71,44 @@ const Dropzone: React.FC<DropzoneProps> = ({ setPdfUrl, isSidebarVisible }) => {
+      formData.append("file", selectedFile);
+  
+      const res = await fetch(`${import.meta.env.VITE_AI_URL}/api/upload`, {
+        method: "POST",
+        body: formData,
+        signal: controller.signal,
+      });
+  
+      const data = await res.json();
+      if (!res.ok) throw new Error(data.error || "Upload failed");
+      setPdfUrl(data.url);
+    } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)